website/www/site/content/en/contribute/_index.md

```diff
@@ -57,46 +57,98 @@ There are lots of opportunities to contribute. You can for example:
  https://cwiki.apache.org/confluence/display/BEAM/Contributor+FAQ)
  - organize local meetups of users or contributors to Apache Beam
 
+Below is a tutorial for contributing code to Beam, covering our tools and typical process in
+detail.
+
 </div>
 
-<a class="arrow-list-header" data-toggle="collapse" href="#collapseContributing" role="button" aria-expanded="false" aria-controls="collapseContributing">
-   {{< figure src="/images/arrow-icon_list.svg">}}
+## Get Started
 
-## Contributing code
+### Connect with the Beam community
 
-  </a>
+1. Consider [subscribing to the dev@beam.apache.org](/community/contact-us/) mailing list, especially if you plan to make more than one change or the change will be large. All decisions are consensus-based and happen on the public mailing list.
+1. (Optionally) Join the [#beam channel of the ASF slack](https://beam.apache.org/community/contact-us/).
+1. Create an account on [Beam issue tracker (JIRA)](https://issues.apache.org/jira/projects/BEAM/issues)
+   (anyone can do this).
 
-<div class="collapse dont-collapse-sm" id="collapseContributing">
+### Accounts and Permissions
 
-Below is a tutorial for contributing code to Beam, covering our tools and typical process in
-detail.
+- [Beam issue tracker (JIRA)](https://issues.apache.org/jira/projects/BEAM/issues):
+  Anyone can access it and browse issues. Anyone can register an account and login
+  to create issues or add comments. Only contributors can be assigned issues. If
+  you want to be assigned issues, a PMC member can add you to the project contributor
+  group.  Email the [dev@beam.apache.org](/community/contact-us)
+  to ask to be added as a contributor in the Beam issue tracker, and include your ASF Jira username.
+
+- [Beam Wiki Space](https://cwiki.apache.org/confluence/display/BEAM/Apache+Beam):
+  Anyone has read access. If you wish to contribute changes, please create an account and request edit access on the
+  [dev@beam.apache.org](https://beam.apache.org/community/contact-us) mailing list (include your Wiki account user ID).
+
+- Pull requests can only be merged by a
+  [Beam committer](https://home.apache.org/phonebook.html?pmc=beam).
+
+- [Voting on a release](https://www.apache.org/foundation/voting.html): Everyone can vote. Only
+  [Beam PMC](https://home.apache.org/phonebook.html?pmc=beam) members should mark their votes as binding.
+
+### Communication
+
+All communication is expected to align with the [Code of Conduct](https://www.apache.org/foundation/policies/conduct).
+
+Discussion about contributing code to Beam happens on the [dev@beam.apache.org mailing list](/community/contact-us/). Introduce yourself!
```

Comment:
change link to https://beam.apache.org/community/contact-us/

Reply:
done